### PR TITLE
AzureEnhancedMonitor V2

### PR DESCRIPTION
**Pull request by v-zhongz**
1.	Switch to LinuxDiagnostic extension version 2.0
-Switched.

2.	Add support for table endpoint, pass table endpoint to LinuxDiagnostic extension as param.
-Updated the node js script.

a.	Add/modify perf counters, for details, see the email thread bellow
-Extension status has been displayed on Azure Portal. And It is checked by Handler, if just check it in PrefCounters, you can't judge whether it is dead status or not. So I don't think it should be write in PrefCounters.

b.	Add an perf counter(\config\errors) for error code
-Added. By record the latest error in disk.

c.	\network({0})\Network Read Bytes and \network({0})\Network Write Bytes are reported per NIC instance. Network adapter must be of Ethernet type, other NICs are filtered out.
Before the counters were cumulative values (sum) for all NICs.
-For each AdapterId, Network Read/Write Bytes are given.

d.	NIC instance counters \network({0})\Minimum Network Bandwidth and \network({0})\Maximum Network Bandwidth are removed, and replaced by counters for VM bandwidth:
\network\VM Minimum Network Bandwidth and \network\VM Maximum Network Bandwidth
-Return hardcoded value is 1000 Mbit/s.

### CODE REVIEW

File: AzureEnhancedMonitor/ext/aem.py [hunk 1]
```diff
@@ -355,20 +383,30 @@ def __init__(self):
...
+                self.networkReadBytes[nicName] = stat[1]
+                self.networkWriteBytes[nicName] = stat[0]
 
     def getAdapterIds(self):
         return self.nicNames
 
-    def getNetworkReadBytes(self):
-        return self.readBytes
+    def getNetworkReadBytes(self, adapterId):
+	if adapterId in self.networkReadBytes :
```
Comment (yuezh) [inline AzureEnhancedMonitor/ext/aem.py]: Indenting issue. Please expand tab into 4 spaces.

File: AzureEnhancedMonitor/ext/aem.py [hunk 2]
```diff
@@ -355,20 +383,30 @@ def __init__(self):
...
+		return self.networkReadBytes[adapterId]
+	else:
```
Comment (yuezh) [inline AzureEnhancedMonitor/ext/aem.py]: Same indenting issue.

File: AzureEnhancedMonitor/ext/aem.py [hunk 3]
```diff
@@ -61,6 +61,32 @@ def get_host_base_from_uri(blob_uri):
 AzureEnhancedMonitorVersion = "1.0.0"
```
Comment (yuezh) [inline AzureEnhancedMonitor/ext/aem.py]: Update to 2.0.0

**Comment by v-zhongz** [comment]
Indentation issues have been revised.

**Pull request merged**